### ISSUE TITLE
filter-menu-button: Missing outline in first and last element in Safari 

### ISSUE THREAD

**Issue by agrananadosbarrantesebay** [eBay/skin#1967]
### Description:

The issue is only happening on **Safari** : https://opensource.ebay.com/ebayui-core/?path=/story/buttons-ebay-filter-menu-button--standard, the focus outline is missing in the first and last element of the menu.

### Analisys: 

@agliga Mentioned this:  "The issue is with the overflow hidden and border radius. That transform should pull it out into another layer."

This code fixes the issue: 
`.filter-menu-button__items {
  transform: translateZ(0);
}`

### Evidence:

![ebay-filter-menu-button-bug](https://user-images.githubusercontent.com/121587009/209888459-5d026177-347c-4595-9d25-3ce38c53720d.gif)

![ebay-filter-menu-button-bug-2](https://user-images.githubusercontent.com/121587009/209888463-57d26cb2-ddec-43f7-a23d-ae8c30db22cc.gif)
